### PR TITLE
[WFLY-6539] Enable http trace requests in testsuite configurations.

### PR DESCRIPTION
**Pull request by istraka**
JIRA: https://issues.jboss.org/browse/WFLY-6539

### CODE REVIEW

**Comment by ctomc** [comment]
this was already addressed in https://github.com/wildfly/wildfly/pull/8720 and https://github.com/jbossas/jboss-eap7/pull/394

**Comment by istraka** [comment]
No, this is for extended tests.

**Comment by istraka** [comment]
But this is for extended tests.

**Comment by ctomc** [comment]
this is ok to test

**Comment by istraka** [comment]
Failed test has nothing to do with enabling trace request.

File: testsuite/integration/src/test/scripts/clustering-build.xml [hunk 1]
```diff
@@ -94,14 +94,26 @@
...
+        <antcall target="enable-http-trace">
+            <param name="cacheMode" value="SYNC" />
+            <param name="stack" value="udp"/>
+        </antcall>
         <antcall target="abstract-build-clustering">
             <param name="cacheMode" value="ASYNC"/>
             <param name="stack" value="udp"/>
         </antcall>
+        <antcall target="enable-http-trace">
+            <param name="cacheMode" value="ASYNC" />
```
Comment (rhusar) [inline testsuite/integration/src/test/scripts/clustering-build.xml]: no reason for the extra space before /> (3 occurrences above)

**Comment by rhusar** [comment]
Otherwise 👍  thanks

**Comment by bstansberry** [comment]
Closing as it has been superceded.

**Pull request closed without merging**